From 074bebf45e265b40598d584f6c82b312e01e0cc1 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 16 Jan 2024 14:34:50 +0100
Subject: [PATCH 0491/1695] plat-stm32mp1: default enable
 CFG_DRIVERS_*_PRINT_TREE

Default enable CFG_DRIVERS_REGULATOR_PRINT_TREE and
CFG_DRIVERS_CLK_PRINT_TREE upon CFG_WITH_STATS as the feature
allows to print the device trees through the statistics PTA
service.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 972eb716601b1ec84b3316aa20f1ac7b83c7a1ee plat-stm32mp1: default enable CFG_DRIVERS_*_PRINT_TREE]
Change-Id: I14f27da3ba29e96ec9dbe40c33c2f2e1b457e220
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index fa90e79ce..c012ed6ea 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -379,6 +379,8 @@ endif
 CFG_ENABLE_EMBEDDED_TESTS ?= y
 CFG_WITH_STATS ?= y
 CFG_WERROR ?= y
+CFG_DRIVERS_CLK_PRINT_TREE ?= $(CFG_WITH_STATS)
+CFG_DRIVERS_REGULATOR_PRINT_TREE ?= $(CFG_WITH_STATS)
 
 # Enable OTP update with BSEC driver
 CFG_STM32_BSEC_WRITE ?= y
-- 
2.47.2

